portfolio/src/main/webapp/style.css

```diff
@@ -24,47 +24,43 @@ li a {
   background-color: #9FA0C3;
   border-bottom-color: #3C617C;
   border-bottom-style: solid;
-  display: block;
   color: black;
-  font-family: monospace;
+  display: block;
```

Comment:
nit: `<div>` elements are displayed as blocks by default, so this line can be removed, unless I'm overlooking some other rule that would be trying to set `display` to something else.